temoa/model_checking/commodity_graph.py

```
@@ -228,6 +235,114 @@ def generate_commodity_graph(
     return dg, sector_colors
 
 
+logger = logging.getLogger(__name__)
+
+
+def generate_graphviz_files(
+    period: Period,
+    region: Region,
+    network_data: NetworkModelData,
```

Ruff (0.14.5)

Unused function argument: `network_data`

(ARG001)